test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -92,7 +98,51 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 					Expect(out).To(Equal(activation_file))
 				}
 			})
+
+			It("[test_id:73046] Verify ovn kube node pod have their cpuset.cpus set to all available cpus", func() {
+				ovnKubenodepod, err := ovnCnfNodePod(ctx, workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+				containerIds, err := ovnPodContainers(&ovnKubenodepod)
+				for _, ctn := range containerIds {
+					var containerCgroupPath string
+					pid, err := nodes.ContainerPid(ctx, workerRTNode, ctn)
+					Expect(err).ToNot(HaveOccurred())
+					cmd := []string{"cat", fmt.Sprintf("/rootfs/proc/%s/cgroup", pid)}
+					out, err := nodes.ExecCommandOnMachineConfigDaemon(context.TODO(), workerRTNode, cmd)
+					Expect(err).ToNot(HaveOccurred())
+					controllers := bytes.Split(out, []byte("\n"))
+					cgroupPathOfPid, err := cgroup.PidParser(out)
+					if len(controllers) > 2 {
+						controller := filepath.Join(cgroupRoot, "/cpuset")
+
+						containerCgroupPath = filepath.Join(controller, cgroupPathOfPid)
+					} else {
+						containerCgroupPath = filepath.Join(cgroupRoot, cgroupPathOfPid)
+					}
+					cmd = []string{"cat", fmt.Sprintf("%s", filepath.Join(containerCgroupPath, "/cpuset.cpus"))}
+					cpus, err := nodes.ExecCommandOnNode(ctx, cmd, workerRTNode)
+					containerCpuset, err := cpuset.Parse(cpus)
+					Expect(containerCpuset).To(Equal(onlineCPUSet), "Burstable pod containers cpuset.cpus do not match total online cpus")
+				}
+
+			})
+
+			It("[test_id:73047] check all pods of ovnkube-node have the right affinity", func() {
+				testutils.KnownIssueJira("OCPBUGS-30806")
```

Comment:
seems that https://issues.redhat.com/browse/OCPBUGS-30806 is verified.   we can probably move this skip

Reply:
Addressed this in the latest commit